tier1-icons.html

```diff
@@ -293,7 +293,7 @@
         <g id="quizzing">
```

Comment:
I think quizzing should probably be mirrored, but the question mark causes a problem. Can we redesign the icon to be more RTL-friendly?